distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java

```diff
@@ -480,6 +486,214 @@ public void testBasicRestAccess() throws Exception {
         getSecurityPolicy().waitForGuestAuthReady(url);
     }
 
+    @Test(expected = NoHttpResponseException.class)
+    public void testBasicAuthOverHttp() throws Exception {
+        String url = SERVICE_ROOT.getUrl() + "/catalog/query?q=*&src=local";
+
+        //try basic auth over http
+        try {
+            configureRestForBasic(SDK_SOAP_CONTEXT);
+            waitForSecurityHandlers(url);
+
+            given().auth()
+                    .basic("admin", "admin")
+                    .when()
+                    .get(url.replace("https://", "http://"));
+        } finally {
+            configureRestForGuest(SDK_SOAP_CONTEXT);
+            getSecurityPolicy().waitForGuestAuthReady(url);
+        }
+    }
+
+    @Test
+    public void testTLSv11() throws Exception {
```

Comment:
❓ If those tests are making sure that basic auth works using different versions of TLS, we may want to add that information to the method names.